assets/ds-moon-preset.js

```diff
@@ -319,6 +319,15 @@ module.exports = {
         "popo": {
           DEFAULT: withOpacityValue("--popo"),
         },
+        "jiren": {
+          DEFAULT: withOpacityValue("--jiren"),
+        },
+        "heles": {
+          DEFAULT: withOpacityValue("--heles"),
+        },
+        "zeno": {
+          DEFAULT: withOpacityValue("--zeno"),
+        },
         black: "#000",
         hover: "rgba(34, 34, 37, 0.12)",
```

Comment:
as far as i remeber we wanted to remove these two lines - hover & primary-hover. their usages (only one as i remeber) should be removed also

Reply:
These legacy classes remain for the time being, with additional explanatory comment.